config/environments/production.rb.template

```diff
@@ -108,6 +108,8 @@ Autolab3::Application.configure do
       exception_recipients: "TEAM@YOURAPP.COM"
     }
 
+  # Feature flag for docker image upload
+  config.docker_image_upload_enabled = false
```

Comment:
Autograder settings page will not load if users do not define this config flag.
![Screenshot 2024-02-08 at 01 07 38](https://github.com/autolab/Autolab/assets/9074856/e82d79cf-566b-4bd9-9fa0-51a40783a7bf)

Suggest using `config.x` namespace instead (see github config at the bottom), since this does not seem to throw an error.